db/schema.rb

```diff
@@ -280,7 +280,7 @@
     t.integer "post_type_id", null: false
     t.text "body_markdown"
     t.integer "answer_count", default: 0, null: false
-    t.datetime "last_activity", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "last_activity", default: -> { "current_timestamp()" }, null: false
```

Comment:
[**chore:**](https://conventionalcomments.org/) Please revert this manually. This change happens automatically sometimes, but we don't want it.

Reply:
Eh, non-blocking. They're both equivalent as far as MySQL is concerned.